src/love.cpp

```diff
@@ -141,6 +141,17 @@ enum DoneAction
 	DONE_RESTART,
 };
 
+static void print_usage()
+{
+	printf("usage:\n"
+		"    love --version                  prints LÖVE version and quits\n"
```

Comment:
Maybe each description sentence/line should end with a period (`.`) - although I don't know if it's common to not do that for this type of thing.

Reply:
I looked at a few linux commands' usage and I think none of them had the period at the end of each line. Some had capital first letter, most had lowercase.